compiler/test/dotc/pos-test-pickling.blacklist

```diff
@@ -37,6 +37,7 @@ i9999.scala
 12093.scala
 9757.scala
 9890.scala
+13491.scala
```

Comment:
How come?

Reply:
The pickling test failure can be minimized to the following: 

```scala
object Test {
  type M[X] = X match { case Int => String }
  def a: Any { type Out = M[Int] } = a
}
```

The failure comes from the refinement type, which is printed as
`scala.Any{Out = scala.String}` before pickling, and
`scala.Any{Out = Test.M[scala.Int]}` after pickling...

I'll open a new issue from that minimization.